packages/common/src/search/_internal/hubSearchEventAttendees.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  GetRegistrationsParams,
+  IGetRegistrationsParams,
+} from "../../events/api";
+import {
+  getRegistrations,
+  IPagedRegistrationResponse,
+} from "../../events/api/orval/api/orval-events";
+import { eventAttendeeToSearchResult } from "../../events/api/utils/search";
+import { IPredicate, IQuery } from "../types/IHubCatalog";
+import { IHubSearchOptions } from "../types/IHubSearchOptions";
+import { IHubSearchResponse } from "../types/IHubSearchResponse";
+import { IHubSearchResult } from "../types/IHubSearchResult";
+
+export const processGetRegistrationsParams = (
+  options: IHubSearchOptions,
+  query: IQuery
+): GetRegistrationsParams => {
+  // valid eys to translate from IQuery predicates
+  const validPredicateKeys: Array<keyof IPredicate> = [
+    "eventId",
```

Comment:
Mentioned in another comment, but `eventId` will be passed in the `options` instead of `query`.